test/Microsoft.AspNetCore.Mvc.Test/Microsoft.AspNetCore.Mvc.Test.csproj

```diff
@@ -2,8 +2,13 @@
 
   <PropertyGroup>
     <TargetFrameworks>$(StandardTestTfms)</TargetFrameworks>
+    <PreserveCompilationContext>true</PreserveCompilationContext>
```

Comment:
Why is this change needed?

Reply:
The one test that I moved uses the deps file to determine if the ReferenceAssemblies list is accurate.